components/network/isc-dhcp/patches/002-dhcp_broadcast_offer_bug.patch

```diff
@@ -0,0 +1,179 @@
+This patch file was developed in-house to fix a Solaris specific bug. We are
+working with upstream, but it is unclear at this point whether or not they are
+going to accept it. This is a highly visible bug which many customers have
+encountered.
+
+diff -r 1680283c6725 common/socket.c
```

Comment:
The patch makes sense. From what I see, this patch inserts arp entry in OS table before sending unicast BOOTP/DHCP response.